operations_on_datastructures/reverse_a_queue_using_a_stack.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file 
+ * @author Alec Rothenberg 
+ * @brief Reverse a queue using a stack
+ * @version 0.1
+ * @date 2022-01-21
+ * 
+ * @copyright Copyright (c) 2022
+ * 
+ */
+
+#include <cassert>
+#include <queue> 
+#include <stack>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```